app/services/medical_professionals/update_service.rb

```diff
@@ -0,0 +1,14 @@
+class MedicalProfessionals::UpdateService
+  class << self
+    def update_doctor_or_mhp(first_name, last_name)
+      doctor = Doctor.find_by(first_name: first_name, last_name: last_name)
+      mhp = MentalHealthProfessional.find_by(first_name: first_name, last_name: last_name)
+
+      if doctor
+        doctor.update(vetted: true)
```

Comment:
This is probably enough for mvp however we'll want to expand on this so that other attrs can be updated and so a resource can be switched from vetted to unvetted.

Reply:
> This is probably enough for mvp however we'll want to expand on this so that other attrs can be updated and so a resource can be switched from vetted to unvetted.

I def agree! I'm happy to revisit this down the line.